### ISSUE TITLE
[plugin-site-issues] production not bumped to v0.2.0

### ISSUE THREAD

**Issue by dduportal** [jenkins-infra/helpdesk#2880]
- Version [ `v0.2.0`](https://github.com/jenkins-infra/docker-plugin-site-issues/releases/tag/v0.2.0) was published (following @halkeye's work + and fixing the issues in #2877 and #2876): 
  - Tag: https://github.com/jenkins-infra/docker-plugin-site-issues/releases/tag/v0.2.0
  - DockerHub: https://hub.docker.com/layers/plugin-site-issues/jenkinsciinfra/plugin-site-issues/v0.2.0/images/sha256-5825505473c10c7944df3ca3819eb3ec65eb55b42153ca070eb6bd4cbc148e8c?context=explore

- BUT it's not picked and updated by the automation system on jenkins-infra/kubernetes-management.

=> Infra team is looking on this to unblock Gavin.

**Comment by dduportal** [comment]
- Automation system correctly detects the new version and propose to update it when in "dry-run" mode: no configuration has to be done on jenkins infra
- However: https://github.com/updatecli/updatecli/issues/632 is bug (currently being fixed by @lemeurherve ) on updatecli v0.23.x. Upcoming 0.23.2 will fix and ensure that PR is opened.

**Comment by dduportal** [comment]
Closing in favor of #2882
